scissors/build.gradle

```diff
@@ -31,7 +31,7 @@ dependencies {
     // Picasso
     provided 'com.squareup.picasso:picasso:[2.4.0, 2.5.2)'
     // Glide
-    provided 'com.github.bumptech.glide:glide:[3.5.0, 3.7.0)'
+    provided 'com.github.bumptech.glide:glide:4.1.1'
```

Comment:
Just a comment: we will have to update our major version of Scissors since we're dropping Glide3 in backward incompatible way